eventstore/tasks.py

```diff
@@ -532,3 +532,49 @@ def process_ada_assessment_notification(
     triage.risk = triage.calculate_risk()
     triage.full_clean()
     triage.save()
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def send_whatsapp_media_template(wa_id, template_name, media_url):
+    headers = {
+        "Authorization": "Bearer {}".format(settings.TURN_TOKEN),
+        "Accept": "application/vnd.v1+json",
+        "Content-Type": "application/json",
+    }
+
+    filename = media_url.split("/")[-1]
```

Comment:
Could maybe use a combination of `urlparse(media_url).path` and `os.path.basename` instead, to handle all of the edge cases (eg. query strings), but this is good too since we'll be controlling the URL.